### PR TITLE
[UUI Theming]: add new components to uui

### PR DESCRIPTION
**Pull request by dsv-sergey**
Add PickerInput, DataTable, ScrollBars, VirtualList, Modals, DropMarker, Tooltip, update Checkbox, Panel, i18n, ColorPicker, remove unused var files, updating the formula for calculating column widths, add top/bottom shadow to ScrollBars

### CODE REVIEW

File: loveship/assets/theme.css [hunk 1]
```diff
@@ -351,6 +354,119 @@
     --uui-tag-count: var(--uui-text-secondary);
     --uui-tag-label: var(--uui-text-secondary);
     --uui-tag-border-radius: 30px;
+
+    /* DropMarker vars */
+    --uui-drop-marker-offset: -1px;
+    --uui-drop-marker-width: 2px;
```
Comment (AlekseyManetov) [inline loveship/assets/theme.css:360]: move this size variables to uui-components

File: loveship/assets/theme.css [hunk 2]
```diff
@@ -351,6 +354,119 @@
...
+    --uui-picker-toggler-text-disabled: var(--uui-control-text-disabled);
+    --uui-picker-toggler-icon: var(--uui-control-icon);
+    --uui-picker-toggler-icon-disabled: var(--uui-control-icon-disabled);
+    --uui-picker-toggler-icon-clickable: var(--uui-icon);
+    --uui-picker-toggler-icon-clickable-hover: var(--uui-icon-hover);
+    --uui-picker-toggler-border-radius: var(--uui-border-radius);
+
+    /* DataTable vars */
+    --uui-dt-cell-border-width: 1px;
+    --uui-dt-cell-editor-border-width: 1px;
```
Comment (AlekseyManetov) [inline loveship/assets/theme.css:393]: move this size variables to uui-components

File: loveship/assets/theme.css [hunk 3]
```diff
@@ -351,6 +354,119 @@
...
+    --uui-dt-row-border: var(--gray30);
+    --uui-dt-row-border-width: 1px;
+}
+
+.uui-theme-loveship .uui-focus {
+    --uui-dt-cell-editor-border-width: 2px;
+}
+.uui-theme-loveship .uui-table-row {
+    --uui-drop-marker-offset: -1px;
+}
```
Comment (AlekseyManetov) [inline loveship/assets/theme.css:453]: move this size variables to uui-components

File: loveship/components/tables/ColumnHeaderDropdown/SortingPanel.scss [hunk 4]
```diff
@@ -1,19 +1,5 @@
 @use '../../../assets/styles/scss/index'  as *;
 
-.filter-sort-button {
```
Comment (AlekseyManetov) [inline loveship/components/tables/ColumnHeaderDropdown/SortingPanel.scss:3]: why we remove it?

File: uui-components/src/table/DataTableRowContainer.tsx [hunk 5]
```diff
@@ -31,16 +31,16 @@ function getSectionStyle(columns: DataColumnProps[], minGrow = 0) {
     let width = 0;
 
     columns.forEach(column => {
-        const columnWidth = (typeof column.width === 'number' ? column.width : (column.minWidth || 0));
+        const columnWidth = (typeof column.width === 'number' ? (column.fix ? column.width : column.width - 1) : (column.minWidth || 0));  // (column.width - 1) do not forget the negative margin of the scrolling columns in the calculation of the width
```
Comment (AlekseyManetov) [inline uui-components/src/table/DataTableRowContainer.tsx:34]: move border width value into const variable and use it here and assign it to css-variable

File: epam-promo/components/pickers/PickerInput.tsx [hunk 6]
```diff
@@ -81,12 +79,6 @@ export class PickerInput<TItem, TId> extends PickerInputBase<TItem, TId, PickerI
             : <DataPickerFooter { ...footerProps } size={ this.props.size } />;
     }
 
-    renderNoFound(props: { search: string, onClose: () => void }) {
```
[the remark this answers is not in the document]
Comment (dsv-sergey) [reply epam-promo/components/pickers/PickerInput.tsx:84]: redundant code, duplicate behavior of DataPickerBody https://github.com/epam/UUI/blob/f1410bd507279b02598400f897313257f736220d/epam-promo/components/pickers/DataPickerBody.tsx#L21

**Pull request merged**